scripts/build-export.ts

```diff
@@ -0,0 +1,113 @@
+import changeCase from 'change-case'
+import fs from 'fs/promises'
+import glob from 'glob'
+import path from 'path'
+import { transform } from '@svgr/core'
+import { transformSync } from 'esbuild'
+
+const DIRECTORY_SOURCE = './src'
+const DIRECTORY_OUTPUT = './dist'
+
+const optimisedComponentTemplate = (
+  { props, imports, componentName, jsx },
+  { tpl }
+) => tpl`${imports};export const ${componentName} = (${props}) => ${jsx}`
+
+const svgoConfig = {
+  multipass: true,
+  plugins: [
+    {
+      name: 'preset-default',
+      params: {
+        overrides: {
+          removeViewBox: false,
+          removeUselessStrokeAndFill: false
+        }
+      }
+    },
+    { name: 'removeAttrs', params: { attrs: 'aria-labelledby' } },
+    { name: 'removeDimensions' },
```

Comment:
is this correct?  do we want to remove the dimensions? I thought we had specific values for that

Reply:
We're setting the height and width in the `Icon` component so they're unnecessary in the source SVG

Reply:
oh I didn't realise